### PR TITLE
Renamed graphql-12.x-plugin to graphql-12.x-15.x-plugin

### PR DESCRIPTION
**Pull request by BFergerson**
Looking into creating additional support and wondering why it kept using version 9 instrumentation.

Calling this `graphql-12.x-scenario` doesn't make much sense btw since it actually tests 12-15.

### CODE REVIEW

**Comment by BFergerson** [comment]
Went ahead and renamed plugin/scenario in preparation for adding additional plugin/scenario.

File: apm-sniffer/apm-sdk-plugin/graphql-plugin/graphql-12.x-15.x-plugin/src/main/resources/skywalking-plugin.def [hunk 1]
```diff
@@ -14,4 +14,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-graphql=org.apache.skywalking.apm.plugin.graphql.v12.define.GraphqlInstrumentation
+graphql=org.apache.skywalking.apm.plugin.graphql.v12tov15.define.GraphqlInstrumentation
```
Comment (wu-sheng) [inline apm-sniffer/apm-sdk-plugin/graphql-plugin/graphql-12.x-15.x-plugin/src/main/resources/skywalking-plugin.def:17]: Should we change the key?

Comment (wu-sheng) [reply apm-sniffer/apm-sdk-plugin/graphql-plugin/graphql-12.x-15.x-plugin/src/main/resources/skywalking-plugin.def:17]: Like Vertx plugin, we have `vertx-core-3.x` for version 3.x. It is better we could have graphql version in the key. Notice, once you change, the Plugin-list.md should be changed too.

Comment (wu-sheng) [inline apm-sniffer/apm-sdk-plugin/graphql-plugin/graphql-12.x-15.x-plugin/src/main/resources/skywalking-plugin.def:17]:
```suggestion
graphql-12.x-15.x=org.apache.skywalking.apm.plugin.graphql.v12tov15.define.GraphqlInstrumentation
```

**Review by wu-sheng — DISMISSED** [review]
LGTM, waiting for tests passed.

File: docs/en/setup/service-agent/java-agent/Plugin-list.md [hunk 2]
```diff
@@ -25,6 +25,7 @@
 - feign-pathvar-9.x
 - finagle
 - graphql
```
Comment (wu-sheng) [inline docs/en/setup/service-agent/java-agent/Plugin-list.md:27]:
```suggestion
```

I think you forgot to remove this.

**Review by wu-sheng — APPROVED** [review]

File: docs/en/setup/service-agent/java-agent/Plugin-list.md (hunk 2 above)
Comment (BFergerson) [reply docs/en/setup/service-agent/java-agent/Plugin-list.md]: There is still the older versions, graphql 8 and 9

File: docs/en/setup/service-agent/java-agent/Plugin-list.md [hunk 3]
```diff
@@ -24,7 +24,7 @@
 - feign-default-http-9.x
 - feign-pathvar-9.x
 - finagle
-- graphql
```
Comment (wu-sheng) [inline docs/en/setup/service-agent/java-agent/Plugin-list.md:27]: I removed this, but it breaks the CI. Maybe there are some other plugin keys should be updated?

File: docs/en/setup/service-agent/java-agent/Plugin-list.md (hunk 2 above)
Comment (BFergerson) [reply docs/en/setup/service-agent/java-agent/Plugin-list.md]: Guessing you would like those keys updated too?

**Review by wu-sheng — APPROVED** [review]

**Pull request merged**